scripts/get-time.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+if [ -z $(which time) ]; then
+    UNAME=$(uname)
+    if [ "$UNAME" == "Linux" ]; then
+        sudo apt-get install time
+    elif [ "$UNAME" == "Darwin"]; then
+        brew install time
+    else
+        sudo apt-get install time
+    fi
+fi
```

Comment:
Could this be moved to a prereq / docker image setup step? Seems a bit extreme do this on the fly (especially on a real machine).

Reply:
The docker image is drawn from a set list of Microsoft-maintained images for use with codespaces, and so hard and/or unsafe to mess with (depending on whether we want to try to convince them to add it to the base image or go it alone with our own image, respectively). There's a module you can include for adding some common utilities, but `time` isn't in that set.

There are triggers for running commands while the container is brought up, and I tried all three of those variants with an install command for time. None of them had any effect; I'm not sure why.

If there's some other option, I'd try it out, but I concluded that there just wasn't any good option and so added this.

Reply:
Fixed in .devcontainer/Dockerfile